harvest/batch-import-marc.sh

```diff
@@ -109,20 +116,33 @@ then
   }
 else
   function log {
-    local FILE=$1
-    cat -u - > $BASEPATH/log/`basename $FILE`.log
+    local FILES=$@
+    local LOGFILE
+    if [ $MAX_BATCH_COUNT -eq "1" ]
+    then
+      LOGFILE=$BASEPATH/log/`basename $FILES`.log
+      > $LOGFILE
+    else
+      TIMESTAMP=$( date +%Y%m%d%H%M%S )
+      LOGFILE=$BASEPATH/log/$TIMESTAMP.log
```

Comment:
I was thinking of using grep, with -l or -m to avoid scanning a lot of large log files. But I would be OK with using the first filename to begin the log file name.